owl/EASE-ACT.owl

```diff
@@ -195,11 +195,35 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Actuating -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Actuating">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <ease:ELANName>move-object</ease:ELANName>
+        <ease:ELANUsageGuideline>Superconcept for the Agent moving some (usually, but not always, grasped) object; use more specific labels where appropriate.</ease:ELANUsageGuideline>
+        <rdfs:comment>Tasks where the goal is to move an object.
```

Comment:
This is very different to how Oxford defines it:

**actuation**
_The action of causing a machine or device to operate._

In that definition actuation does not depend on any motion.

Reply:
I'd rather look at definitions of "actuate" (the verb).

wiktionary.org:  To activate, or to put into motion; to animate.
merriam-webster: to put into mechanical action or motion

Reply:
ok